cli/tests/integrations/test_package.py

```diff
@@ -332,19 +306,6 @@ def test_install_bad_package_version():
 def test_package_metadata():
     _install_helloworld()
 
-    # test marathon labels
-    expected_metadata = b"""eyJkZXNjcmlwdGlvbiI6ICJFeGFtcGxlIERDT1MgYXBwbGljYX\
-Rpb24gcGFja2FnZSIsICJtYWludGFpbmVyIjogInN1cHBvcnRAbWVzb3NwaGVyZS5pbyIsICJuYW1l\
-IjogImhlbGxvd29ybGQiLCAicGFja2FnaW5nVmVyc2lvbiI6ICIyLjAiLCAicG9zdEluc3RhbGxOb3\
-RlcyI6ICJBIHNhbXBsZSBwb3N0LWluc3RhbGxhdGlvbiBtZXNzYWdlIiwgInByZUluc3RhbGxOb3Rl\
-cyI6ICJBIHNhbXBsZSBwcmUtaW5zdGFsbGF0aW9uIG1lc3NhZ2UiLCAidGFncyI6IFsibWVzb3NwaG\
-VyZSIsICJleGFtcGxlIiwgInN1YmNvbW1hbmQiXSwgInZlcnNpb24iOiAiMC4xLjAiLCAid2Vic2l0\
-ZSI6ICJodHRwczovL2dpdGh1Yi5jb20vbWVzb3NwaGVyZS9kY29zLWhlbGxvd29ybGQifQ=="""
-
-    expected_command = b"""eyJwaXAiOiBbImRjb3M8MS4wIiwgImdpdCtodHRwczovL2dpdGh\
-1Yi5jb20vbWVzb3NwaGVyZS9kY29zLWhlbGxvd29ybGQuZ2l0I2Rjb3MtaGVsbG93b3JsZD0wLjEuM\
-CJdfQ=="""
```

Comment:
Can we just update this base64 string with whatever Cosmos is writing which may include the nulls.